### PR TITLE
repeat and bounce

### PR DESCRIPTION
**Pull request by callumok2004**
finally a pull request that doesnt have a load of file changes I DIDNT DO reeeeeee

### CODE REVIEW

File: websites/B/Bounce/dist/metadata.json [hunk 1]
```diff
@@ -11,12 +11,14 @@
...
+    "itsbounce.uk",
+    "wearebounce.co.uk"
   ],
-  "version": "2.5.3",
-  "logo": "https://boun.cc/cdn/noshadow.png",
-  "thumbnail": "https://i.imgur.com/XmJTiZY.png",
+  "version": "3.0.0",
+  "logo": "https://i.imgur.com/4tsFh1F.png",
+  "thumbnail": "https://i.imgur.com/tlLbekj.png",
   "color": "#32a852",
```
Comment (Bas950) [inline websites/B/Bounce/dist/metadata.json:22]: get a better thumbnail

File: websites/R/Repeat/dist/metadata.json [hunk 2]
```diff
@@ -12,8 +12,8 @@
     "repeatradio.net",
     "repeat.pw"
   ],
-  "version": "1.0.2",
-  "logo": "https://i.imgur.com/Mh7vi7e.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/I5G1Lvn.png",
   "thumbnail": "https://i.imgur.com/7RjRna8.png",
```
Comment (Bas950) [inline websites/R/Repeat/dist/metadata.json:17]: 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/Bounce/dist/metadata.json (hunk 1 above)
Comment (callumok2004) [reply websites/B/Bounce/dist/metadata.json:22]: there
is
nothing
better

File: websites/B/Bounce/dist/metadata.json [hunk 3]
```diff
@@ -17,8 +17,8 @@
     "wearebounce.co.uk"
   ],
   "version": "3.0.0",
-  "logo": "https://i.imgur.com/4tsFh1F.png",
-  "thumbnail": "https://i.imgur.com/tlLbekj.png",
+  "logo": "https://i.imgur.com/ZXbdFGq.png",
+  "thumbnail": "https://i.imgur.com/oebZGyW.png",
```
Comment (StrikerFRFX) [inline websites/B/Bounce/dist/metadata.json:21]: Mouse cursorrrrrrrrrrr

File: websites/R/Repeat/dist/metadata.json [hunk 4]
```diff
@@ -12,8 +12,8 @@
     "repeatradio.net",
     "repeat.pw"
   ],
-  "version": "1.0.2",
-  "logo": "https://i.imgur.com/Mh7vi7e.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/ufIgYNI.png",
   "thumbnail": "https://i.imgur.com/7RjRna8.png",
```
Comment (StrikerFRFX) [inline websites/R/Repeat/dist/metadata.json:17]: Needs to be a screenshot of the site or something suitable for a thumbnail

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]

File: websites/R/Repeat/dist/metadata.json (hunk 4 above)
Comment (StrikerFRFX) [inline websites/R/Repeat/dist/metadata.json:17]: Change this thumbnail to something similar to the Bounce one.

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]

File: websites/R/Repeat/dist/metadata.json (hunk 4 above)
Comment (callumok2004) [reply websites/R/Repeat/dist/metadata.json:17]: it was there before, untouched by me, not my problem

**Review by Bas950 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by StrikerFRFX — APPROVED** [review]

**Pull request merged**